core/arch/opnd.h

```diff
@@ -996,6 +1074,22 @@ extern const reg_id_t dr_reg_fixer[];
 #    define REG_XMM13 DR_REG_XMM13
 #    define REG_XMM14 DR_REG_XMM14
 #    define REG_XMM15 DR_REG_XMM15
+#    define REG_XMM16 DR_REG_XMM16
```

Comment:
We could drop this no-prefix support for new registers.

Reply:
I was wondering why this was there.

Reply:
I would vote for not putting new ones here and documenting that by the DynamoRIO_REG_COMPATIBILITY in DynamoRIOConfig.cmake.in.

Reply:
Needs https://github.com/DynamoRIO/dynamorio/pull/3503.